src/aqdata/aq_base.cc

```diff
@@ -39,8 +39,8 @@ void AQBase<dim>::InitRefBCInd () {
   // uses deal.II generated mesh or mesh from gmsh with proper
   // boundary IDs setup: {0,1,2,3,4,5} for {xmin,xmax,ymin,ymax,zmin,zmax}
   if (have_reflective_bc_) {
-    Assert (dim>1,
-            dealii::ExcNotImplemented());
+    AssertThrow (dim>1,
```

Comment:
Hmm, the reflective boundary is actually implemented for general dimension and should work in 1D, so please delete this but maybe add a `TODO` comment for testing for it.

Reply:
Exception removed and test added.